Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListContentController/ConversationListContentController.swift

```diff
@@ -96,6 +99,12 @@ final class ConversationListContentController: UICollectionViewController, Popov
         }
     }
 
+    @objc
+    func showErrorAlertForConversationRequest() {
+        typealias ErrorString = L10n.Localizable.Error
+        UIAlertController.showErrorAlert(message: ErrorString.Conversation.missingLegalholdConsent)
```

Comment:
The typealias isn't so necessary when you only use it once.

```suggestion
        UIAlertController.showErrorAlert(message: L10n.Localizable.Error.Conversation.missingLegalholdConsent)
```